examples/user_guide/Pandas_API.ipynb

```diff
@@ -1058,9 +1081,7 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "scrolled": true
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
     "# Doesn't currently work as expected. Work-around:\n",
```

Comment:
What is the workdaround here? Seems like a comment that needs deleting? Also "Colormaps can also be used other plot types" should maybe be "Colormaps can also be used with other plot types", but GH won't let me edit that since it presumably predates this PR.

Reply:
Yes it's unrelated to this PR. I will open a new PR with changes to the docs.

Reply:
https://github.com/holoviz/hvplot/pull/678